src/client/lib/seo-head.js

```diff
@@ -12,16 +12,19 @@
  * @param {Number} [seo.image.width]
  * @param {Number} [seo.image.height]
  */
-export default function seoHead ({ title, description, image }) {
+export default function seoHead ({ title, description, image, url }) {
   return {
     title,
+    link: [
+      url && { rel: 'canonical', href: url },
+    ].filter(Boolean),
```

Comment:
Could you add a small comment after this explaining why? `// only pass valid <link> objects to template` or something.